common/constants/big.go

```diff
@@ -12,12 +12,25 @@ var (
 	Big0 = big.NewInt(0)
 	// Big1 is 1 represented as type "math/big"
 	Big1 = big.NewInt(1)
+	// Big2Exp232 is 2^232 respresented as type "math/big"
```

Comment:
Its not just for clarity but it also helps us to avoid allocating new memory for the same values over and over again throughout the code as they are pointers.